### ISSUE TITLE
Bug: no-void should have option to allow ignoring promises

### ISSUE THREAD

**Issue by marcospgp** [eslint/eslint#19876]
### Environment

Node version:
npm version:
Local ESLint version:
Global ESLint version:
Operating System:

### What parser are you using?

Default (Espree)

### What did you do?

<details>
<summary>Configuration</summary>

```js
<!-- Paste your configuration here -->
```
</details>

```js
<!-- Paste your code here -->
```

### What did you expect to happen?

no-void would be configurable per title

### What actually happened?

isn't

### Link to Minimal Reproducible Example

docs

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by mdjermanovic** [comment]
Thanks for the issue, but as stated in [the docs](https://eslint.org/docs/latest/rules/no-void), this rule is currently [frozen](https://eslint.org/docs/latest/contribute/core-rules#frozen-rules) and is not accepting feature requests.
